config/peerpods/podvm/bootc/Containerfile.rhel

```diff
@@ -0,0 +1,82 @@
+# build pause
+FROM registry.k8s.io/pause:3.9 as pause-bin
```

Comment:
Hi @snir911 ,

An user building a custom operator's built podvm can customize the pause image by setting the `PAUSE_IMAGE_REPO`, `PAUSE_IMAGE_VERSION` and `CLUSTER_PULL_SECRET_AUTH_FILE` variable. Would make sense to pass these as build arguments in this containerfile too?

Apart from that, maybe we want to keep the pause image version in sync with the operator's built podvm, i.e,  https://github.com/openshift/sandboxed-containers-operator/blob/devel/config/peerpods/podvm/lib.sh#L8

Example:
```
ARG PAUSE_IMAGE_REPO="quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256"
ARG PAUSE_IMAGE_VERSION="7f3cb6f9d265291b47a7491c2ba4f4dd0752a18b661eee40584f9a5dbcbe13bb"

FROM ${PAUSE_IMAGE_REPO}:${PAUSE_IMAGE_VERSION} as pause-bin
```

Reply:
OCP's pause image is much more heavy so i preferred the k8s one (before we couldn't use that)

about the AUTH_FILE this is indeed needed, I will address in separate

Reply:
I think this won't be needed as the global authfile is passed and soon we'll have also support in imagePullSecret for custom secrets

Reply:
The imagePullSecret is not for the pause image. Note that the secret that we use here is for downloading and embedding the pause image.